packages/components/src/local/coa-status-board/index.tsx

```diff
@@ -28,10 +28,10 @@ export const CoaStatusBoard: React.FC<Props> = ({ templates, messages, channel,
   const [filteredRows, setFilterdRows] = useState<Row[]>([])
   const [inFilterMode, setFilterMode] = useState<boolean>(false)
 
-  const myParticipations = channel.participants.filter((p) => p.force === role.forceName && ((p.roles.includes(role.roleId)) || p.roles.length === 0))
```

Comment:
Hello @lebaphi - I'm pretty sure we can just delete/deprecate this component, and any custom children it has. The data model doesn't work for it, but it has been superceded by `coa-status-board-2`.  Deleting it will prevent us wasting time in having to fix/maintain compiler issues.

Reply:
Drop this comment - I'm deleting the component.